SeaPublicWebsite/Models/EnergyEfficiency/ReturnToActionPlanViewModel.cs

```diff
@@ -8,4 +8,5 @@ public class ReturnToActionPlanViewModel
     [GovUkValidateRequired(ErrorMessageIfMissing = "Enter a valid email address")]
     public string EmailAddress { get; set; }
     public bool EmailSent { get; set; }
+    public bool PdfVersion { get; set; }
```

Comment:
Not sure I like `PdfVersion` as a variable name, I keep thinking it will be a version number rather than a boolean. Would `IsPdf` work? Or do you have any other ideas?